Rewatch.lua

```diff
@@ -7,7 +7,7 @@
 -- Please give full credit when you want to redistribute or modify this addon!
 
 
-local rewatch_versioni = 60007;
+local rewatch_versioni = 80000;
```

Comment:
There is no need to keep Rewatch version aligned with WoW version :-D you just released 2 complete rework versions of Rewatch (7.0 and 8.0) 😀 🤣

Reply:
(I choose to release 61000 for the BFA + Shaman support release)

Reply:
Woops xD. I'll take a look at the 6100 release.